node/cmd/ccq/devnet.config.json

```diff
@@ -0,0 +1,64 @@
+{
+  "permissions": [
+    {
+      "userName": "Test User",
+      "apiKey": "my_secret_key",
+      "allowedCalls": [
+        {
+          "ethCall": {
+            "note:": "Name of WETH on Goerli",
+            "chain": 2,
+            "contractAddress": "B4FBF271143F4FBf7B91A5ded31805e42b2208d6",
```

Comment:
Do the contract addresses need to have leading "0x"?

Reply:
Nope. It handles both ways.